File: app/artifact_agent/app/src/generated_outputs/java/calculator_api/src/main/java/com/example/calculatorapi/model/CalculationResponse.java
package com.example.calculatorapi.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.Instant;

@Data
@AllArgsConstructor
public class CalculationResponse {
    private String id;
    private double operand1;
    private double operand2;
    private String operation;
    private double result;
    private Instant createdAt;
}

File: app/artifact_agent/app/src/generated_outputs/.net/calculator_api/CalculatorApi/Models/CalculationResponse.cs
namespace CalculatorApi.Models;

public class CalculationResponse
{
    public string Id { get; set; }
    public double Operand1 { get; set; }
    public double Operand2 { get; set; }
    public string Operation { get; set; }
    public double Result { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}